### ISSUE TITLE
Only folders showing files

### ISSUE THREAD

**Issue by herlon214** [nikolaydubina/go-cover-treemap#8]
It's probably a bug, once the only folders shouldn't show any file.
If you take a look into the example in the README: https://github.com/nikolaydubina/go-cover-treemap/blob/main/docs/hugo-only-folders-no-aggregation.svg

You can see it's still showing files, for instance: `resources/images/exif/exif.go`

**Comment by nikolaydubina** [comment]
yeah, that is just example when no aggregation. real version (in main) is not running that anyways. still thanks for pointing out. fixed.
